ykman-gui/main.cpp

```diff
@@ -60,11 +61,22 @@ int main(int argc, char *argv[])
 
     engine.load(QUrl(url_prefix + main_qml));
 
-    if (argc > 2 && strcmp(argv[1], "--log-level") == 0) {
-        if (argc > 4 && strcmp(argv[3], "--log-file") == 0) {
-            QMetaObject::invokeMethod(engine.rootObjects().first(), "enableLoggingToFile", Q_ARG(QVariant, argv[2]), Q_ARG(QVariant, argv[4]));
+    QCommandLineParser parser;
+    parser.setApplicationDescription("Cross-platform application for YubiKey configuration");
+    parser.addHelpOption();
+    parser.addVersionOption();
+    parser.addOptions({
+        {"log-level", QCoreApplication::translate("main", "Set log level to <LEVEL>"), QCoreApplication::translate("main", "LEVEL")},
```

Comment:
Maybe print out the different log-levels, similar to what `ykman -h` does?

Reply:
Good idea! I'll see if Qt has an API for that, or if we'll have to do it by hand.

Reply:
There seems to be no API for allowed values, so I just hard-coded them into the option description. They're unlikely to change anyway.